draft-packages/menu/KaizenDraft/Menu/Menu.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from "@kaizen/draft-button"
+import { cleanup, fireEvent, render } from "@testing-library/react"
+import * as React from "react"
+import Menu from "./Menu"
+
+afterEach(cleanup)
+
+describe("Dropdown", () => {
+  test("renders default view", () => {
```

Comment:
nit/ In other tests, we tend to use `it` rather than `test`

Reply:
Happy to change this after merging if that's OK with you. These tests were copied from the existing Dropdown and I just removed a few that were no longer applicable.

Reply:
(The merge gives me access to the MenuItemProps which unblocks other work)